wrappers/wrapper.rb

```diff
@@ -1360,7 +1360,8 @@ def simplify_service_setup_duration_and_vehicle_setup_modifiers(vrp, solution =
         }
       when :rewind
         # take it back in case in dicho and there will be re-optimization
-        return nil unless vrp.services.any?{ |s| s.activity[:simplified_setup_duration] }
+        return nil if vrp.services.any?{ |s| s.activities.any? } ||
```

Comment:
We need to do the same skip in `:patch_solution` stage as well, no?

Reply:
A solution stop only reference a single activity from the original service